.github/workflows/propose_translation.yml

```diff
@@ -0,0 +1,66 @@
+name: Propose Translation
+on:
+  workflow_call:
+    inputs:
+      branch:
+        description: 'The branch to checkout and run the action in'
+        type: string
+        default: ${{ github.ref_name }}
+      python_version:
+        description: 'The python version to use'
+        type: string
+        default: "3.8"
+      language:
+        description: 'The language to translate to (ISO 3166-1 alpha-2 code; form "xx-xx")'
+        type: string
+        default: ""
+      locale_folder:
+        description: 'The base folder to look for localization files'
+        type: string
+        default: "locale"
+      reviewers:
+        description: 'A comma or newline-separated string of reviewers'
+        type: string
+        default: "emphasize"
+      
+
+jobs:
+  Propose_translation:
+    env:
+      TARGET_LANG: ${{ inputs.language }}
+      API_KEY: ${{ secrets.DL_API_KEY }}
+      LOCALE_FOLDER: ${{ github.workspace }}/action/package/${{ inputs.locale_folder }}
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ inputs.branch }}
+        path: action/package/
+    - name: Checkout Scripts Repo
+      uses: actions/checkout@v4
+      with:
+        repository: OpenVoiceOS/.github
+        path: action/github/
+    - name: Setup Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{ inputs.python_version }}
+    - name: Install Translate Tools
+      run: |
+        python -m pip install -r action/github/requirements/translation.txt
+    - name: Run Translate Script
+      run: |
+        python action/github/scripts/translate.py
+        cd ${{ github.workspace }}/action/package/
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v5
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        commit-message: autotranslate
+        author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+        title: Proposed ${{ inputs.language }} Translations
+        body: Translations for review
+        labels: translation
+        branch: staging/translation_${{ inputs.language }}
+        reviewers: ${{ inputs.reviewers }}
```

Comment:
I'm not sure if YAML will accept raw newline-delimited strings as a value without using `|` or similar

Reply:
i don't see problems in `"1\n2\n3\n"`

Reply:
Good point, it wouldn't render in a way that should cause YAML any problems.